ProcessMaker/Listeners/CommentsSubscriber.php

```diff
@@ -26,11 +27,26 @@ public function onActivityCompleted(ActivityCompletedEvent $event)
         $user_id = $token->user ? $token->user_id : null;
         $user_name = $token->user ? $token->user->fullname : __('The System');
 
+        // Check the authenticated user
+        $executer_user_id = null;
+        $executer_user = __('The System');
+        if (Auth::check()) {
+            $executer_user_id = Auth::user() ? Auth::user()?->id : null;
+            $executer_user = Auth::user() ? Auth::user()?->fullname : __('The System');
```

Comment:
I changed the previous two for the following; it’s clearer and doesn’t repeat the variable initialization.
`           $user = Auth::user();`
`            if ($user) {`
`               $executer_user_id = $user->id;`
`                $executer_user = $user->fullname;`
`            }`